astron/config/cluster.yml

```diff
@@ -79,9 +79,8 @@ roles:
         min: 100000000
         max: 399999999
       backend:
-        type: mongodb
-        server: 127.0.0.1:21021
-        database: test
+        type: yaml
```

Comment:
misread: that is a good point.. we should at least get that backed out and back to the yaml version so that people can run private servers off the TTFS tree easier.